conjure-generator-common/src/main/java/com/palantir/conjure/CaseConverter.java

```diff
@@ -99,6 +101,8 @@ private static Case nameCase(String name) {
                 return nameCase;
             }
         }
-        throw new IllegalArgumentException("Unexpected case for: " + name);
+        throw new IllegalArgumentException("Unexpected case for: '" + name
+                + "'. Expected one to be in one of these case formats: "
+                + Arrays.stream(Case.values()).map(Enum::name).collect(Collectors.toList()));
```

Comment:
Could we convert each element from screaming snake into itself for additional inline docs? e.g. `[lowerCamelCase, kebab-case, snake_case]`

Reply:
We could also hard code the string, either way works for me :-]

Reply:
I had originally wanted to avoid that so the message got automatically updated for any new entries to the enum, but lost that when going to Guava.  So we might as well go all the way and just hardcode the string!